lib/supa/builder.rb

```diff
@@ -21,6 +21,19 @@ def initialize(subject, representer:, tree:)
       end
     end
 
+    def attributes(*args)
+      options = if args[-1].is_a?(Hash)
```

Comment:
@goozzik Let's change the main interface from the old school, to ruby 2.x because it doesn't make sense to suffer without reason

1. Good new **kvargs `define_method(command) do |name, **options, &block|`
2. Let's have a normal `attributes` class with it's own behaviour
3. Let's implement `attributes` as multiple-times `attribute` without options hack, but just passing them into single case

Because now solution look dirty and hacky and don't follow the logic created before